tests/test_snippets.py

```diff
@@ -33,8 +33,11 @@ def test_snippets_rst(
     builder.src_path.joinpath("index.rst").write_text(content, encoding="utf8")
     write_assets(builder.src_path)
     builder.build()
+    pformat = builder.get_doctree("index").pformat()
+    # fixed in https://github.com/executablebooks/MyST-Parser/pull/465
+    pformat = pformat.replace('<bullet_list bullet="-">', "<bullet_list>")
     file_regression.check(
-        builder.get_doctree("index").pformat(),
+        pformat,
```

Comment:
I'm glad to see this wasn't a fault on my end (unless I'm misunderstanding this change).

Reply:
No just a minor difference between RST and MyST AST